app/models/manageiq/consumption/showback_event.rb

```diff
@@ -24,7 +24,7 @@ class ManageIQ::Consumption::ShowbackEvent < ApplicationRecord
 
   include_concern 'CPU'
   include_concern 'MEM'
-
+  include_concern 'TEMPLATE'
```

Comment:
TEMPLATE should be changes to another name (perhaps VM_TEMPLATE). It is not clear defined as tempalte

Suggestions:
ENVELOPE
RESOURCES_REQUESTED
FLAVOR